docs/user-guide/deployments/configuration.md

```diff
@@ -550,6 +550,20 @@ In the configuration files of standalone GreptimeDB, the value needs to be set a
 mode = "standalone"
 ```
 
+### Heartbeat configuration
+
+```toml
+[heartbeat]
+interval = "3s"
+retry_interval = "3s"
+```
+
+| Key                        | Type   | Default | Description                                                                                                                                        |
+|----------------------------|--------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| `heartbeat`                | --     | --      | --                                                                                                                                                 |
+| `heartbeat.interval`       | String | `3s`    | Interval for sending heartbeat messages to the Metasrv.                                                                                            |
+| `heartbeat.retry_interval` | String | `3s`    | Interval for retrying to establish the heartbeat connection to the Metasrv. Note that this option is ignored in Datanode heartbeat implementation. |
```

Comment:
```suggestion
| `heartbeat.retry_interval` | String | `3s`    | Interval for retrying to establish the heartbeat connection to the Metasrv. Note that this option is ignored in Datanode heartbeat implementation because the Datanode must renew its lease through heartbeat within the keep-alive mechanism's lease period. It has a special retry strategy and doesn't allow custom configuration.|
```